src/websys_bridge.rs

```diff
@@ -364,6 +364,16 @@ pub fn node_from_ws<Ms: Clone>(node: &web_sys::Node) -> Option<Node<Ms>> {
                 });
             el.attrs = attrs;
 
+            // todo etc
+            let svg_tags = vec![
+                "svg", "circle", "line", "rect"
+            ];
+            let svg_tags: Vec<String> = svg_tags.into_iter().map(|t| t.to_string()).collect();
```

Comment:
We can inline it and crate only one Vec:
```rust
let svg_tags: Vec<String> = [
    "svg", "circle", "line", "rect"
].iter().map(ToString::to_string).collect();
```

Reply:
Update: I think we don't need to create `Vec` at all, 
```rust
let svg_tags = [
    "svg", "circle", "line", "rect"
];
```
should be enough.

Reply:
Had that originally, but couldn't get contains to work with it due to a double ref; might be another way.

Reply:
Running into issues with `contains` and double-references.

Reply:
I'm able to compile this:
```rust
    let svg_tags = [
        "svg", "circle", "line", "rect"
    ];
    let tag_name = "xxx".to_string();
    if svg_tags.contains(&tag_name.to_lowercase().as_str()) {
//        el.namespace = Some(Namespace::Svg);
    }
```
It should help you.

Reply:
That worked

Reply:
I don't like the idea of repeating the tags since it makes things more complicated when updating them, and is more error-prone, but I don't see an obvious way around it here.